pkg/plugins/common/kustomize/v1/scaffolds/internal/templates/config/certmanager/certificate.go

```diff
@@ -49,6 +50,11 @@ const certManagerTemplate = `# The following manifests contain a self-signed iss
 apiVersion: cert-manager.io/v1
 kind: Issuer
 metadata:
+  labels:
+    app.kubernetes.io/name: selfsigned-issuer
+    app.kubernetes.io/component: certificate
+    app.kubernetes.io/created-by: {{ .ProjectName }}
+    app.kubernetes.io/part-of: {{ .ProjectName }}
```

Comment:
By looking at : https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/
I understand that the part-of, in this case, would be cert-manager.io

Also, we can add

app.kubernetes.io/managed-by -> kustomize
app.kubernetes.io/version -> v1

@everettraven wdyt?

Reply:
Hi @everettraven, 

I review the doc and your comments 
-  part-of : By looking at the examples I think you are right that needs to be the Operator itself for all manifests that are part-of the operator. However, in this case they are NOT/ They are helper to install the certmanager. So, the name of a higher level application is cert-manager.io. Note that some manifests scaffolds are not part at all of the Operator itself and these ones are the only case scenarios that ihmo we cannot add project as a value.
 
-  managed-by: See the doc description. `The tool is used to manage the operation of an application. Also, see in the examples that you will find in all of the helm, because helm is the tool used to manage the manifest in our case all that is config/ is managed by kustomize. Then, we need to think should we use kustomize for the config and what we add for the manifests created in the reconciliation? So, here I think could also fit kubebuilder. See that `cert-manager.io.` is not a tool. cert-manager is a higher application that if the operator want uses certified requires to have installed. 

- app.kubernetes.io/version -> If the manifest is part-of the Operator itself then I agree that the version can be the operator version. However, if the higher application is not the Operator itself and the manifest scaffold here is ONLY a helper to have a pre-requirement installed then I think we need to have the version of the higher application for that.

Reply:
Hi @everettraven ,

So from this one:

```
  labels:
    app.kubernetes.io/name: serving-cert
    app.kubernetes.io/component: certificate
    app.kubernetes.io/created-by: {{ .ProjectName }}
    app.kubernetes.io/part-of: {{ .ProjectName }}
```

Alll the above shows fine. **And I agree we are adding a certificate for the project itself.** 

We just not added  the following:
    app.kubernetes.io/version: "5.7.21" -> in our case the version of the Operator itself 
    app.kubernetes.io/managed-by: helm -> in our case we can use kustomize or kubebuilder 
 
So, I think we can leave this empty and also decide in a follow up